spyder/plugins/ipythonconsole.py

```diff
@@ -932,7 +933,8 @@ def write_to_stdin(self, line):
     @Slot(bool)
     @Slot(str)
     @Slot(bool, str)
-    def create_new_client(self, give_focus=True, filename=''):
+    def create_new_client(self, give_focus=True, filename='',
+                          stderr_dir=programs.TEMPDIR):

```

Comment:
Please don't add a new parameter here because we'd need to also add new `Slot`'s